concepts/05 Widgets/Scheduler/020 Data Binding/10 Simple Array/15 ArrayStore.md

```diff
@@ -114,7 +179,91 @@ Data kept in the **ArrayStore** can be processed in the [DataSource](/api-refere
         [dataSource]="appointmentDataSource">
     </dx-scheduler>
 
+##### Vue
 
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :data-source="dataSource"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxScheduler from 'devextreme-vue/scheduler';
+
+    const appointments = [{ 
+        desc: 'Meet with a customer', 
+        employee: "Mary Watson",
+        from: new Date(2016, 4, 10, 11, 0), 
+        to: new Date(2016, 4, 10, 13, 0) 
+    }, 
+    // ...
+    ];
+
+    export default {
+        components: {
+            DxScheduler
+        },
+        data() {
+            return {
+                dataSource: new DevExpress.data.DataSource({
```

Comment:
These notes apply to all examples in this PR:
https://github.com/DevExpress/devextreme-documentation/pull/344#discussion_r380198535
https://github.com/DevExpress/devextreme-documentation/pull/344#discussion_r380198759